### DISCUSSION TITLE
Flux1 support

### DISCUSSION THREAD

**Discussion by portu-sim** [portu-sim/comfyui_bmab#34] [Announcements]
BMAB now supports Flux 1.

<img width="1717" alt="image" src="https://github.com/user-attachments/assets/dfa36eb1-6978-47e7-a0d0-67c82639cf52">

**Comment by kinizwei** [comment]
"BMAB Flux Integrator" Node Missing ?? All other Nodes are available

**Comment by portu-sim** [comment]
Please check errors.
if you installl comfyui clean, you install blendmodes.
I'll update soon.


2025년 2월 14일 (금) 오후 9:02, kinizwei ***@***.***>님이 작성:
